webapp/first_snap/content/ros/build.yaml

```diff
@@ -0,0 +1,54 @@
+linux:
+  auto:
+    - action: Return to the root directory of the project containing your snapcraft.yaml and run snapcraft
+      command: snapcraft
+    - action: You will find a .snap file in the same directory that you ran the snapcraft command. You can inspect the contents to ensure it contains all of the application's assets
+      command: unsquashfs -l *.snap
+    - action: Install the snap
+      command: sudo snap install --devmode --dangerous *.snap
+    - action: List your installed snaps to confirm
+      command: snap list
+    - action: Run the snap
+      command: test-ros-talker-listener-{name} -h
+    - warning: |
+        Make sure the name matches what has been used previously (i.e <code>test-ros-talker-listener-{name}</code>)
+  manual:
+    - action: Return to the root directory of the project containing your snapcraft.yaml and run the snapcraft Docker image
+      command: docker run -v $PWD:$PWD -w $PWD snapcore/snapcraft snapcraft
```

Comment:
As in ROS2, this won't work.

Reply:
If you choose macOS in the guide, the `manual` option is not shown, so the docker usage is not presented to the user. @evandandrea do we want to do a site-wide cleanup of the manual configuration option?

Reply:
Ignore the Docker stuff. I'll work on a PR to drop that from all the first snap flows.